terragraph_planner/data/template.yaml

```diff
@@ -112,189 +110,150 @@ LINE_OF_SIGHT:
   # distance is computed to be the point at which the link capacity goes to 0
   # (ignoring interference) based on the provided radio equipment
   # specifications. Increasing the minimum MCS shortens that distance.
-
   MINIMUM_MCS_OF_BACKHAUL_LINKS: null
   MINIMUM_MCS_OF_ACCESS_LINKS: null
 
-  # [Bool] Enable an ellipsoidal LOS model
-
+  # [Bool] Enable the ellipsoidal LOS model
   USE_ELLIPSOIDAL_LOS_MODEL: False
 
-  # [Float] The radius of the Fresnel Zone at its
-  # widest point. The radio MUST be in [0, 1]. Not required if ellipsoidal
-  # LOS model is enabled.
-
+  # [Float] The radius of the Fresnel Zone at its widest point. The radius must
+  # be in [0, 1]. Not used if the ellipsoidal LOS model is enabled.
```

Comment:
I forgot why I put this sentence ("The radius must be in [0, 1].") here, but it should be removed. Fresnel should be positive, but there's no upper bound, although usually a fresnel radius of 60 GHz radio should be around 1 m.